<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RemoteClass</class>
 <widget class="QMainWindow" name="RemoteClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>960</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>960</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Remote</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>961</width>
      <height>431</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>10</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SetNoConstraint</enum>
     </property>
     <item>
      <widget class="QTableWidget" name="Server_TableWidget_Online">
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QTableWidget" name="Server_TableWidget_Message">
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>960</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="Server_Menu">
    <property name="title">
     <string>菜单</string>
    </property>
    <addaction name="Server_Menu_Set"/>
    <addaction name="Server_Menu_Exit"/>
    <addaction name="Server_Menu_Add"/>
   </widget>
   <addaction name="Server_Menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="Server_Menu_Set">
   <property name="text">
    <string>本地设置</string>
   </property>
  </action>
  <action name="Server_Menu_Exit">
   <property name="text">
    <string>退出程序</string>
   </property>
  </action>
  <action name="Server_Menu_Add">
   <property name="text">
    <string>添加信息（测试）</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Remote.qrc"/>
 </resources>
 <connections/>
</ui>
